src/components/Avatar/Avatar.component.tsx

```diff
@@ -363,7 +357,7 @@ const Avatar: FC<AvatarProps> = ({
               />
             )}
             {effects?.vignette && <Vignette eskil={false} offset={0.5} darkness={0.5} />}
-            <BrightnessContrast brightness={0.025} contrast={0.25} />
```

Comment:
Currently bugged alongside the ao causing white flickering, will see if there is a fix available rather than outright removing

Reply:
Reintroduced with a small increase instead.